version.go

```diff
@@ -15,14 +16,25 @@ var (
 	version = "(devel)" // to match the default from runtime/debug
 )
 
-func printVersion() {
+func gofumptVersion() string {
 	// don't overwrite the version if it was set by -ldflags=-X
-	if info, ok := debug.ReadBuildInfo(); ok && version == "(devel)" {
+	if version != "(devel)" {
+		return version
+	}
+	if testVersion := os.Getenv("GOFUMPT_VERSION_TEST"); testVersion != "" {
+		version = testVersion
+		return testVersion
+	}
+	if info, ok := debug.ReadBuildInfo(); ok {
```

Comment:
I think it would be easier if you kept the old logic as-is, and just unserted the `os.Getenv(...) != nil` at the top. The priority doesn't matter too much, because the env var is only used in the tests, where it's the only thing that matters. For real users, where ldflags and ReadBuildInfo matters, then the env var will never exist. (Or, if they're setting it, that's kinda their own fault)